images/airflow-managed-services/Dockerfile

```diff
@@ -1,14 +1,15 @@
 ARG BASE_IMAGE=quay.io/cloud-bulldozer/airflow:2.3.1
+# Hypershift Compilation
+FROM golang:1.18 AS hypershift
+RUN git clone --branch main https://github.com/openshift/hypershift
+WORKDIR hypershift
+RUN make build
+# Runtime image
 FROM ${BASE_IMAGE} as runtime
 USER root
 RUN curl -L https://go.dev/dl/go1.18.2.linux-amd64.tar.gz -o go1.18.2.linux-amd64.tar.gz
 RUN tar -C /usr/local -xzf go1.18.2.linux-amd64.tar.gz
 ENV PATH=$PATH:/usr/local/go/bin
-RUN mkdir -p /home/airflow/go
-ENV GOPATH=/home/airflow/go
-RUN go install github.com/markbates/pkger/cmd/pkger@latest
-RUN mv $GOPATH/bin/pkger /usr/local/bin/
-ENV GOFLAGS=
```

Comment:
I  am [working](https://github.com/mukrishn/airflow-kubernetes/commit/27a8ec6120dad51a557c640bdd5bd23f3c53fdf2#diff-2077ba877e816fdbbd1f6ebea9c103d42861a79b2549fb3e93e16ae708c93e01R83-R92) on enabling a `hypershift_cli_version` var to pick latest CLI, so you recommend download and install GO1.18 during dag runtime and make this light weight. I am okay if my understanding is right.

Reply:
Its ok to have it here, so runs will be faster. Executions should consider that any software they need to run is already available on the image.

We have the possibility to download latest Rosa/OCM and now hyper shift on the run time, this is ok during the initial phase, because the development is too fast to generate a new image on every change, but we should consider rely on published releases in the future, as therñy are what customers can consume, we cannot just download latest commit from master, because if the commit is broker, all the execution will be invalid. Default usage should be use the last published version, and use master only when a required feature won't be available on it

Reply:
> but we should consider rely on published releases in the future, as therñy are what customers can consume, we cannot just download latest commit from master, because if the commit is broker, all the execution will be invalid
Yes that is also part of that link, after that we can pin them to a particular release and fork.